README.md

```diff
@@ -4,52 +4,68 @@ This is a **group exercise**, so you should be working in pairs of two students.
 
 The Goal is to **practise writing readable, maintainable and reliable code collaboratively.**
 
```

Comment:
Maybe you could rephrase it, saying this is a practice exams which was created in a seminar by students, but without telling the grading or rephrasing the exact task